Formula/f/fzf-zsh-plugin.rb

```diff
@@ -0,0 +1,32 @@
+class FzfZshPlugin < Formula
+  desc "ZSH plugin to enable fzf search for frequently used tools"
+  homepage "https://github.com/unixorn/fzf-zsh-plugin"
+  url "https://github.com/unixorn/fzf-zsh-plugin/archive/04ae801c5381a61889286345ee763347415d655c.tar.gz"
+  version "2025.05.30"
```

Comment:
we need a stable release, https://docs.brew.sh/Acceptable-Formulae#stable-versions

Reply:
- https://github.com/unixorn/fzf-zsh-plugin/issues/119